sdk/src/metrics/export/periodic_exporting_metric_reader.cc

```diff
@@ -111,7 +118,7 @@ bool PeriodicExportingMetricReader::CollectAndExportOnce()
     status = future_receive.wait_for(std::chrono::milliseconds(export_timeout_millis_));
     if (status == std::future_status::timeout)
     {
-      cancel_export_for_timeout = true;
+      *cancel_export_for_timeout = true;
```

Comment:
```suggestion
      cancel_export_for_timeout->store(true);
```

nit -just for readibility.

Reply:
Thanks and fixed.